packages/consent/consent-tools/src/domain/create-wrapper.ts

```diff
@@ -43,6 +43,12 @@ export const createWrapper = <Analytics extends AnyAnalytics>(
       settings,
       options
     ): Promise<void> => {
+      // page() should happen as early as possible in order to prevent stale page context
```

Comment:
Can you add the comment here that we need to do it for consent before we load settings/destintations hence why we bypass the default behavior

Reply:
Sure!